api/handler/put.go

```diff
@@ -762,6 +747,27 @@ func (h *handler) CreateBucketHandler(w http.ResponseWriter, r *http.Request) {
 	api.WriteSuccessResponseHeadersOnly(w)
 }
 
+func (h handler) setPolicy(prm *layer.CreateBucketParams, locationConstraint string, userPolicies []*accessbox.ContainerPolicy) {
+	prm.Policy = h.cfg.Policy.Default
+
+	if locationConstraint == "" {
+		return
+	}
+
+	if policy, ok := h.cfg.Policy.RegionMap[locationConstraint]; ok {
```

Comment:
Make a comment here about parsing order. It is important to check default policies from config first, because they are controlled by system administrators and they are more relevant than immutable user policies from accessbox.

Reply:
Current order the following (every next override previous on conflict):

* default policy from config
* policies policy from config
* policies from user access box

Is it OK?

Reply:
Oh, I've been mistaken in my first comment. I thought that policies from config have priority over access box, so I suggested to highlight it in the comment.

In fact they didn't.